File: Array/reverseArray.java
// import java.util.*;
class reverseArray{

    public static void reverseArr(int[] arr){
        int start =  0;
        int last = arr.length-1;
        while(start <= last){
            int temp = arr[start];
            arr[start] = arr[last];
            arr[last] = temp;
            start++;
            last--;
        }
        for (int i : arr) {
            System.out.print(i+"  ");
        }
    }
    public static void main(String[] args) {
        int[] array = {1,2,3,4,5};
        reverseArr(array);
    }
}

File: Array/reverseArray.py
def reverseArr(arr):
    start = 0
    last = len(arr)-1
    while(start < last):
        arr[start] , arr[last] = arr[last] , arr[start]
        start +=1
        last -=1
    
    for i in arr: 
        print(" : ",i)
       
array = [10,20,30,40,50,60,70,80]  
reverseArr(array)  